docs/components/Colors.css

```
@@ -0,0 +1,51 @@
+:root {
+  --color-width: 177px;
+  --color-height: 60px;
+  --hex-font-size: 11px;
+  --border-color: #f0f0f0;
+  --color-text-white: #fefefe;
+  --color-text-gray: var(--accent-dark);
+}
+
+.Colors {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 20px var(--space-medium);
+}
+
+.Color {
+  width: var(--color-width);
+  height: var(--color-height);
+  display: inline-block;
+  margin-right: 20px var(--space-medium);
+}
+
+.Color .Color__bg {
+  height: 80%;
+  position: relative;
+  border-radius: 2px;
+}
```

Nit: One advantage of using BEM is that it tries to avoid nesting selectors so that there's no higher specificity getting applied. For this and other styles I think you can just use a single selector.

```suggestion
.Color__bg {
  height: 80%;
  position: relative;
  border-radius: 2px;
}
```